packages/code-du-travail-modeles/src/modeles/conventions/1505_commerces_detail_fruits_legumes/__tests__/indemnite-licenciement/indemnite.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { IndemniteLicenciementPublicodes } from "../../../../../publicodes";
+
+const engine = new IndemniteLicenciementPublicodes(
+  modelsIndemniteLicenciement,
+  "1505"
+);
+
+describe("Calcul de l'indemnité de licenciement CC 1505", () => {
+  test.each`
+    seniorityRight | seniority | expectedCompensation
+    ${6 / 12}      | ${6 / 12} | ${0}
+    ${0.67}        | ${0.67}   | ${435.5}
+    ${3}           | ${6}      | ${3900}
+    ${6}           | ${6}      | ${3900}
+    ${11}          | ${11}     | ${7366.67}
+    ${14}          | ${11}     | ${7366.67}
+    ${20}          | ${20}     | ${15166.67}
```

Comment:
```suggestion
    seniorityRight | seniority | expectedCompensation
    ${6 / 12}      | ${1} | ${0}
    ${8/12}        | ${0.67}   | ${435.5}
    ${8/12}           | ${6}      | ${3900}
    ${8/12}           | ${6}      | ${3900}
    ${8/12}          | ${11}     | ${7366.67}
    ${8/12}          | ${11}     | ${7366.67}
    ${8/12}          | ${20}     | ${15166.67}
```

Je pense que l'on peut challenger les variables avec ces valeurs :)